### PR TITLE
fix bag, refactoring

### PR DESCRIPTION
**Pull request by Shishani58**

### CODE REVIEW

File: Tooba3/UIView.swift [hunk 1]
```diff
@@ -37,3 +38,12 @@ extension UIView {
         }
     }
 }
+
+extension UIView {
+    func roundCorners(corners:UIRectCorner, radius: CGFloat) {
```
Comment (Jihadik) [inline Tooba3/UIView.swift]: Do you use some auto-formatter? If no, let's apply it to the whole code base in a separate PR.
What does this change do?

File: Tooba3/ProjectInfoScrollableViewController.swift [hunk 2]
```diff
@@ -7,343 +7,337 @@
...
+                                            self?.showSelector()
+                                        }
+                    }, completion: nil)
+            } else {
+                self.showSelector()
+            }
+        }
+    }
+    
+    /// Получить информацию о проекте по ID
```
Comment (Jihadik) [inline Tooba3/ProjectInfoScrollableViewController.swift]: Get information about a project by the id.

File: Tooba3/ProjectInfoScrollableViewController.swift [hunk 3]
```diff
@@ -7,343 +7,337 @@
...
+                                guard let self = self else { return }
+                                self.projectInfoResponse(projectInfo)
+            }, onErrorCompletion: { [weak self] in
+                guard let self = self else { return }
+                self.projectInfoResponseError()
+        })
+    }
     
-    @IBOutlet weak var lblReachedValue: UILabel!
+    /// Получить информацию о проекте по tag
```
Comment (Jihadik) [inline Tooba3/ProjectInfoScrollableViewController.swift]: Get information about a project by its tag.

File: Tooba3/ProjectInfoScrollableViewController.swift [hunk 4]
```diff
@@ -7,343 +7,337 @@
...
+            guard let self = self else { return }
+            self.projectInfoResponse(projectInfo)
+            }, NO: { [weak self] in
+                guard let self = self else { return }
+                self.projectInfoResponseError()
+        })
+    }
     
-    @IBOutlet weak var txtDonateValue: CurrencyLabel!
+    /// Обработка ответа при запросе fetchProjectInfo()
```
Comment (Jihadik) [inline Tooba3/ProjectInfoScrollableViewController.swift]: Response handling of fetchProjectInfo() request.

File: Tooba3/ProjectInfoScrollableViewController.swift [hunk 5]
```diff
@@ -7,343 +7,337 @@
...
+    }
     
-    @IBOutlet weak var btnPay: UIButton!
-    @IBOutlet weak var btnShare: UIButton!
-    @IBOutlet weak var lblLikeCount: UILabel!
-    @IBOutlet weak var lblTags: UILabel!
-    @IBOutlet weak var lblDescription: UILabel!
-    @IBAction func btnPlusPressed(_ sender: Any) {
-        self.txtDonateValue.becomeFirstResponder()
+    /// Обработка ошибки при запросе fetchProjectInfo()
```
Comment (Jihadik) [inline Tooba3/ProjectInfoScrollableViewController.swift]: Error handling of fetchProjectInfo() request.

File: Tooba3/UIView.swift (hunk 1 above)
Comment (Shishani58) [reply Tooba3/UIView.swift]: I just moved this code fragment to an extension for UIView, where it should

**Pull request merged**